guides/assets/javascripts/guides.js

```diff
@@ -24,46 +24,74 @@
   }
 
   document.addEventListener("turbo:load", function() {
-    var guidesMenu = document.getElementById("guidesMenu");
-    var guides     = document.getElementById("guides");
+    // The guides menu anchor is overridden to expand an element with the entire
+    // index on the same page. It is important that both the visibility is
+    // changed and that the aria-expanded attribute is toggled.
+    //
+    // Additionally, keyboard users should be able to close these guides by
+    // pressing escape, which is the standard key to collapse expanded elements.
+    var guidesMenuButton = document.getElementById("guidesMenu");
+    var guides = document.getElementById(guidesMenuButton.getAttribute('aria-controls'));
```

Comment:
While we are renaming things here, I think we can probably also rename the id of the button and rename the `guides` to `guidesMenu`?
```suggestion
    var guidesMenuButton = document.getElementById("guidesMenuButton");
    var guidesMenu = document.getElementById(guidesMenuButton.getAttribute('aria-controls'));
```

Reply:
Sure!

I tried not to touch unrelated things except for those that really didn't make sense in context.

Any other renames we should apply right away?

Reply:
I think these are the most obvious ones that are touched by this PR.
Any others I'm missing?

Reply:
I'd opt to rename container to main as it's... not a container. s-hidden also doesn't really mean anything and there is already a different class used to hide things conditionally.

Should I just make changes to improve the maintainability now that we touch these things?